evap/staff/importers.py

```diff
@@ -497,26 +497,51 @@ def write_enrollments_to_db(self, semester, vote_start_datetime, vote_end_date):
             # add all participants at once to create only a single log message
             for evaluation, participants in participants_per_evaluation.items():
                 evaluation.participants.add(*participants)
+            self.create_success_messages(students_created, responsibles_created)
 
-        msg = format_html(
-            _("Successfully created {} courses/evaluations, {} participants and {} contributors:"),
-            len(self.evaluations),
-            len(students_created),
-            len(responsibles_created),
+    def create_success_messages(self, students_created, responsibles_created):
```

Comment:
The name seems a bit off to me: I would assume that a function `create*` returns whatever was created. Maybe something like "add" or "append" would fit better? What do you think?

This only creates one message, so it shouldn't be "messages" either